docs/aws/audit/iamcompliance/rules/root_account_activity_remediation.mdx

```diff
@@ -1,6 +1,82 @@
 
 ### Triage and Remediation
 <Tabs>
+
+<Tab title='Cause'>
+### Check Cause
+<AccordionGroup>
+<Accordion title='Using Console' defaultOpen='true'>
+1. Sign in to the AWS Management Console and open the IAM console at https://console.aws.amazon.com/iam/.
+
+2. In the navigation pane, choose "Credential Report". This report will list all your account's users and the status of their various credentials.
+
+3. If the report is not ready, choose "Download Report". This will generate a new report.
+
+4. Open the report and look for the root account. Check the "password_last_used" or "access_key_1_last_used_date" columns. If these columns show a recent date, it means the root account has been used recently.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. List all IAM users: To check the activity of the root account, you first need to list all IAM users. You can do this by running the command `aws iam list-users`. This will return a list of all IAM users in your AWS account.
+
+3. Check last used access key: To check when the root account was last used, you can use the command `aws iam get-access-key-last-used --access-key-id <access-key-id>`. Replace `<access-key-id>` with the access key ID of the root account. This will return the date and time when the root account was last used.
+
+4. Check CloudTrail logs: AWS CloudTrail logs all API calls made in your AWS account. You can use these logs to monitor the activity of the root account. To do this, you can use the command `aws cloudtrail lookup-events --lookup-attributes AttributeKey=Username,AttributeValue=root`. This will return a list of all API calls made by the root account.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary Python libraries: Before you can start writing the script, you need to install the necessary Python libraries. The AWS SDK for Python (Boto3) allows Python developers to write software that makes use of services like Amazon S3, Amazon EC2, etc. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Configure AWS Credentials: You need to configure your AWS credentials. You can configure credentials by using the AWS CLI or by directly adding them to your script. However, it's not a good practice to hard code your credentials into your script.
+
+```python
+import boto3
+
+aws_access_key_id = 'YOUR_ACCESS_KEY'
+aws_secret_access_key = 'YOUR_SECRET_KEY'
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target="_blank">Click here for more details</a></b>